tools/Makefile.unix

```diff
@@ -421,7 +421,7 @@ ifeq ($(CONFIG_UBOOT_UIMAGE),y)
 		cp -f uImage /tftpboot/uImage; \
 	fi
 endif
-	$(call POSTBUILD)
+	$(call POSTBUILD, $(TOPDIR))
```

Comment:
Similar change needs to be done to the Makefile.win.

Out of curiosity, are you using TOPDIR in your POSTBUILD script to move around the final BIN?

Reply:
> Similar change needs to be done to the Makefile.win.
Got.

> Out of curiosity, are you using TOPDIR in your POSTBUILD script to move around the final BIN?
Yes, just to improve compatibility, postbuild will require different BIN directories in some scenarios on out platform.